sfepy/terms/terms_multilinear.py

```diff
@@ -745,9 +744,11 @@ def make_function(self, texpr, *args, diff_var=None):
             return einfo.eval_einsum
 
         if einfo.eargs is None:
+            equl_args = len(args) == len(self.ats)
```

Comment:
This does not check that args really equal term.args ;)

What about just passing `arg_pos=k` always, and setting the name to `'par{}'.format(arg_pos)`?

Reply:
If `len(args)` is equal to `len(term.args)` then `arg_pos=k` may be reasonable. If not, the argument name shoud be specified by a tuple `(arg, arg_name)`. Otherwise, the name is set to `unknown_parameter`.